zanata-war/src/main/java/org/zanata/rest/service/SourceDocResourceService.java

```diff
@@ -421,7 +421,7 @@ private HLocale validateTargetLocale(LocaleId locale, String projectSlug,
         } catch (ZanataServiceException e) {
             log.warn(
                     "Exception validating target locale {} in proj {} iter {}",
-                    e, locale, projectSlug, iterationSlug);
+                    locale, projectSlug, iterationSlug);
```

Comment:
> http://slf4j.org/faq.html#paramException
> If the exception is not the last argument, it will be treated as a plain object and its stack trace will NOT be printed. **However, such situations should not occur in practice.**

Ha! Obviously that wasn't true in this case.